docs/AppService/PublishAppService/index.md

```diff
@@ -0,0 +1,81 @@
+## PublishAppService
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Registers a service offered by the HMI on the module
+
+### Request
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|appServiceManifest|[Common.AppServiceManifest](../../common/structs/#)|true||
+
+### Response
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|appServiceRecord|[Common.AppServiceRecord](../../common/structs/#)|false||
+
+### Sequence Diagrams
+|||
+PublishAppService
+![PublishAppService](./assets/PublishAppService.png)
+|||
+
+### Example Request
+
+```json
+{
+	"id": 1000,
+	"jsonrpc": "2.0",
+	"method": "AppService.PublishAppService",
+	"params": {
+		"appServiceManifest": {
+			"allowAppConsumers": true,
+			"handledRPCs": [41],
+			"rpcSpecVersion": {
+				"majorVersion": 5,
+				"minorVersion": 1
+			},
+			"serviceName": "IVI_Media",
+			"serviceType": "MEDIA"
+		}
+	}
+}
+```
+
+### Example Response
+
+```json
+{
+   "id" : 1000,
+   "jsonrpc" : "2.0",
+   "result" : {
+      "appServiceRecord" : {
+         "serviceActive" : true,
+         "serviceID" : "1f89547cc0b12d5a52e896c45e141497f7af50e1e2dc8705914e75ef6fbeac03",
+         "serviceManifest" : {
+            "allowAppConsumers" : true,
+            "handledRPCs" : [ 41 ],
+            "rpcSpecVersion" : {
+               "majorVersion" : 5,
+               "minorVersion" : 1
+            },
+            "serviceName" : "IVI_Media",
+            "serviceType" : "MEDIA"
+         },
+         "servicePublished" : true
+      },
+      "code" : 0,
+      "method" : "AppService.PublishAppService"
```

Comment:
Missing closing bracket `}`

Reply:
Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/139/commits/f2ac175098e32dec1463786a71bc5430fc269ae7